corgidrp/l4_to_tda.py

```diff
@@ -142,6 +145,48 @@ def convert_to_flux(input_dataset, fluxcal_factor):
     return flux_dataset
 
 
+def compute_calibrated_contrast_curve(input_dataset, halfwidth=None):
```

Comment:
there should be an option to pass in a requested list of separations. and we do interpolation with the KL_THRU and CT_THRU curves to get those values at the requested separations.